.editorconfig

```diff
@@ -181,3 +181,6 @@ csharp_new_line_before_finally = true
 csharp_new_line_before_members_in_object_initializers = true
 csharp_new_line_before_members_in_anonymous_types = true
 csharp_new_line_between_query_expression_clauses = true
+
+[*.yml]
+quote_type = single
```

Comment:
for some reason VSCode kept replacing single-quotes with double-quotes in the yaml despite all the local setting for using singles. This helped: https://stackoverflow.com/questions/48864985/vscode-single-to-double-quote-automatic-replace#comment105479981_48866406

Reply:
I hope it's alright to keep it here